From 35722d59b2126537669d796533597f156ee1ef47 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Thu, 23 Jul 2020 15:37:25 -0500
Subject: [PATCH 22/23] imx: beacon: imx8mn: Update RAM timings

The Nano running with industrial temperature SoC's needs slightly
different timings in order to operate properly.

This patch updates the timings with the I-Temp settings.  They
should work on commercial temp products as well.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 board/beacon/imx8mn/lpddr4_timing.c | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/board/beacon/imx8mn/lpddr4_timing.c b/board/beacon/imx8mn/lpddr4_timing.c
index e5af4c2935..6b42e8d3d1 100644
--- a/board/beacon/imx8mn/lpddr4_timing.c
+++ b/board/beacon/imx8mn/lpddr4_timing.c
@@ -16,16 +16,16 @@ struct dram_cfg_param ddr_ddrc_cfg[] = {
 	{ 0x3d400304, 0x1 },
 	{ 0x3d400030, 0x1 },
 	{ 0x3d400000, 0xa1080020 },
-	{ 0x3d400020, 0x203 },
+	{ 0x3d400020, 0x202 },
 	{ 0x3d400024, 0x3e800 },
-	{ 0x3d400064, 0x6100e0 },
+	{ 0x3d400064, 0x3000e0 },
 	{ 0x3d4000d0, 0xc003061c },
 	{ 0x3d4000d4, 0x9e0000 },
 	{ 0x3d4000dc, 0xd4002d },
 	{ 0x3d4000e0, 0x310000 },
 	{ 0x3d4000e8, 0x66004d },
 	{ 0x3d4000ec, 0x16004d },
-	{ 0x3d400100, 0x1a201b22 },
+	{ 0x3d400100, 0x1a200d22 },
 	{ 0x3d400104, 0x60633 },
 	{ 0x3d40010c, 0xc0c000 },
 	{ 0x3d400110, 0xf04080f },
@@ -67,15 +67,15 @@ struct dram_cfg_param ddr_ddrc_cfg[] = {
 	{ 0x3d400498, 0x620096 },
 	{ 0x3d40049c, 0x1100e07 },
 	{ 0x3d4004a0, 0xc8012c },
-	{ 0x3d402020, 0x1 },
+	{ 0x3d402020, 0x0 },
 	{ 0x3d402024, 0x7d00 },
 	{ 0x3d402050, 0x20d040 },
-	{ 0x3d402064, 0xc001c },
+	{ 0x3d402064, 0x6001c },
 	{ 0x3d4020dc, 0x840000 },
 	{ 0x3d4020e0, 0x310000 },
 	{ 0x3d4020e8, 0x66004d },
 	{ 0x3d4020ec, 0x16004d },
-	{ 0x3d402100, 0xa040305 },
+	{ 0x3d402100, 0xa040105 },
 	{ 0x3d402104, 0x30407 },
 	{ 0x3d402108, 0x203060b },
 	{ 0x3d40210c, 0x505000 },
@@ -92,7 +92,7 @@ struct dram_cfg_param ddr_ddrc_cfg[] = {
 	{ 0x3d402194, 0x80303 },
 	{ 0x3d4021b4, 0x100 },
 	{ 0x3d4020f4, 0xc99 },
-	{ 0x3d403020, 0x1 },
+	{ 0x3d403020, 0x0 },
 	{ 0x3d403024, 0x1f40 },
 	{ 0x3d403050, 0x20d040 },
 	{ 0x3d403064, 0x30007 },
-- 
2.17.1

